packages/ilib-loctool-webos-c/CFile.js

```diff
@@ -120,9 +120,8 @@ CFile.prototype.parse = function(data) {
     // To extract resBundle_getLocString()
     reGetLocString.lastIndex = 0; // just to be safe
     var result = reGetLocString.exec(data);
-    while (result && result.length > 1 && result[3]) {
-        match = result[3];
-
+    while (result && result.length > 1) {
+        match = result[3] ? result[3]: " "; //Update for the following case support: resBundle_getLocString(_gpstResBundle, "");
```

Comment:
```suggestion
        match = result[3] ? result[3]: ""; //Update for the following case support: resBundle_getLocString(_gpstResBundle, "");
```
That way, the "if" statement on line 125 will fail and it will skip this string entirely.